<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="filenameLabel">
          <property name="text">
           <string>Filename:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="filenameInput"/>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="filenameButton">
          <property name="text">
           <string>Load data</string>
          </property>
          <property name="icon">
           <iconset theme="QIcon::ThemeIcon::DocumentOpen"/>
          </property>
          <property name="shortcut">
           <string>Ctrl+O</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="fromLabel">
          <property name="text">
           <string>From time:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>To time:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDateTimeEdit" name="dateFrom">
          <property name="displayFormat">
           <string>yyyy/MM/dd hh:mm:ss.zzz</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QDateTimeEdit" name="dateTo">
          <property name="dateTime">
           <datetime>
            <hour>22</hour>
            <minute>0</minute>
            <second>0</second>
            <year>2000</year>
            <month>12</month>
            <day>31</day>
           </datetime>
          </property>
          <property name="displayFormat">
           <string>yyyy/MM/dd hh:mm:ss.zzz</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="applyDateFilter">
          <property name="text">
           <string>Apply filter</string>
          </property>
          <property name="icon">
           <iconset theme="QIcon::ThemeIcon::EditFind"/>
          </property>
          <property name="shortcut">
           <string>Ctrl+A</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <widget class="QListWidget" name="masterList">
        <property name="font">
         <font>
          <family>Cascadia Mono</family>
         </font>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="localHostLabel">
             <property name="text">
              <string>Local IP address</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="localHostDisplay">
             <property name="font">
              <font>
               <family>Cascadia Mono</family>
              </font>
             </property>
             <property name="frame">
              <bool>false</bool>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="remoteHostLabel">
             <property name="text">
              <string>Remote IP address</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QLineEdit" name="remoteHostDisplay">
             <property name="font">
              <font>
               <family>Cascadia Mono</family>
              </font>
             </property>
             <property name="frame">
              <bool>false</bool>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="dateLabel">
             <property name="text">
              <string>Date</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QDateEdit" name="dateDisplay">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="font">
              <font>
               <family>Cascadia Mono</family>
              </font>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
             <property name="accelerated">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="timeLabel">
             <property name="text">
              <string>Time</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QTimeEdit" name="timeDisplay">
             <property name="font">
              <font>
               <family>Cascadia Mono</family>
              </font>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
             <property name="accelerated">
              <bool>true</bool>
             </property>
             <property name="displayFormat">
              <string>hh:mm:ss.zzz</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="statusCodeLabel">
             <property name="text">
              <string>Status code</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLCDNumber" name="statusCodeDisplay">
             <property name="lineWidth">
              <number>0</number>
             </property>
             <property name="digitCount">
              <number>3</number>
             </property>
             <property name="segmentStyle">
              <enum>QLCDNumber::SegmentStyle::Flat</enum>
             </property>
            </widget>
           </item>
           <item row="3" column="2">
            <widget class="QLabel" name="methodLabel">
             <property name="text">
              <string>Method</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="3" column="3">
            <widget class="QLineEdit" name="methodDisplay">
             <property name="font">
              <font>
               <family>Cascadia Mono</family>
              </font>
             </property>
             <property name="frame">
              <bool>false</bool>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="resourceLabel">
             <property name="text">
              <string>Resource</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="sizeLabel">
             <property name="text">
              <string>Size</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QLCDNumber" name="sizeDisplay">
             <property name="lineWidth">
              <number>0</number>
             </property>
             <property name="segmentStyle">
              <enum>QLCDNumber::SegmentStyle::Flat</enum>
             </property>
            </widget>
           </item>
           <item row="5" column="2">
            <widget class="QLabel" name="hostnameLabel">
             <property name="text">
              <string>Hostname</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="5" column="3">
            <widget class="QLineEdit" name="hostnameDisplay">
             <property name="font">
              <font>
               <family>Cascadia Mono</family>
              </font>
             </property>
             <property name="frame">
              <bool>false</bool>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="6" column="0">
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Orientation::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="4" column="1" colspan="3">
            <widget class="QLineEdit" name="resourceDisplay">
             <property name="font">
              <font>
               <family>Cascadia Mono</family>
              </font>
             </property>
             <property name="frame">
              <bool>false</bool>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="prevButton">
             <property name="text">
              <string>Previous</string>
             </property>
             <property name="icon">
              <iconset theme="QIcon::ThemeIcon::GoPrevious"/>
             </property>
             <property name="shortcut">
              <string>Ctrl+P</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="nextButton">
             <property name="text">
              <string>Next</string>
             </property>
             <property name="icon">
              <iconset theme="QIcon::ThemeIcon::GoNext"/>
             </property>
             <property name="shortcut">
              <string>Ctrl+N</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
